### PR TITLE
Dev ui fixes

### PR DESCRIPTION
**Pull request by vadnguyen**
responsive tables now has min-width, 
added navbar alignment/padding for mobile, 
added Route Switch to render ErrorView for 404, 
cleaned up no data panels (used Jumbotron styling instead of custom)

Screen Shot 2020-04-06 at 10 59 15 AM
Screen Shot 2020-04-06 at 10 58 58 AM

### CODE REVIEW

**Comment by kkuratsu** [comment]
@vadnguyen Can we make the user menu collapse? I also notice that the site name "Collective Shield" breaks in two lines.

**Comment by mjoyce91** [comment]
I seemed to be running into an issue where nothing would display on login

**Comment by vadnguyen** [comment]
@mjoyce91 - not seeing the blank screen. I used followed the documentation and locally renders fine. Please re-try

**Comment by ConnorDY** [comment]
The navigation bar looks a bit odd and the claim button stretches in the Open Requests table.
Also, I noticed the footer doesn't stick to the bottom the way I'd expect it to.

image

File: ui/src/components/ShippingModal.tsx [hunk 1]
```diff
@@ -19,14 +19,18 @@ const ShippingModal: React.FC = () => {
...
-                Jeffrey@collectiveshield.org
-              </a>{' '}
-              with the requestor's shipping address.
-            </p>
+            <ListGroup as="ol">
+              <ListGroup.Item as="li">
+                Contact the requestor to obtain their shipping address.
+              </ListGroup.Item>
+              <ListGroup.Item as="li">
+                2. Email{' '}
```
Comment (mjoyce91) [inline ui/src/components/ShippingModal.tsx:27]: `1.` was removed but not `2.`

**Pull request merged**